.travis.yml

```diff
@@ -1,10 +1,43 @@
 # See Dockerfile for the full build instructions
 sudo: required
-language: cpp
+language: python
+cache: pip
+
+#mainly for flake8 as everything else runs in docker
+python: 
+  - 3.5
+
+git:
+  quiet: true
```

Comment:
I'm not a huge fan of disabling any output in CI, it makes it much harder to diagnose CI failures. But feel free to override me here :-)